CHANGELOG.md

```diff
@@ -3,6 +3,18 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [0.9.5-beta] - 2016-06-16
+### Added
+- Deploy to App Engine flexible compat environment support.
```

Comment:
I guess nothing was really done explicitly for this so maybe we don't call it out